src/Parallel/Invoke.hpp

```diff
@@ -3,9 +3,36 @@
 
 #pragma once
 
+#include <cstddef>
+#include <type_traits>
 #include <utility>
 
+#include "Parallel/TypeTraits.hpp"
+#include "Utilities/Requires.hpp"
+
 namespace Parallel {
+namespace detail {
+// Allow 64 inline entry method calls before we fall back to Charm++. This is
+// done to avoid blowing the stack.
```

Comment:
This was an amazingly mind blowing question :D In the Charm++ context "inline" means "don't go through the Charm++ runtime system but go straight to the local object if it exists." Other than here, any suggestions on where to best document this? Maybe the parallelization group?

Reply:
I totally missed the relevant text in the PR description, sorry! Okay this makes more sense now. But I'm still not sure what the cap at 64 is for. Is the idea to allow 64 nested local object calls (roughly 64 deep on stack), but then give control back to the runtime system which will be making calls one-by-one and therefore avoiding deep stacking?

Reply:
I would find a comment like this one more useful at the call site (so around the if/else block in `receive_data` below), because then I can better see how it relates to the charm++. But this does mean duplicating the comment at different call sites... perhaps a compromise is to give a slightly more detailed explanation in the Doxygen group, and a brief summary at each call site? What do you think?

Reply:
Okay, added comments :)